### PR TITLE
Adding litercay policy

### PR DESCRIPTION
**Pull request by denispop9**
# Description

Added html to the literacy policy.

## What

Literacy policy now has a blurb and has been converted to html.

## Why

To appear on the pop up called "read me" when clicked.

## How

By adding html.

## Verification Steps

Clone this branch
Make a localhost server in the main directory of the project
Open localhost server and go to About -> Policies
Ensure that the 'Attendance and Participation Policy' Section of the page is filled out correctly.
If it isn't ,request changes.

# Checklist:

- [x] This pr is linked to the related trello card
- [x] My code follows the style guidelines of this project
- [x] I have requested a code review from (at least) two team members
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] This change has been verified by another team member

### CODE REVIEW

**Comment by denispop9** [comment]
@laurafitzgerald @evanhearne can you have a look at this for me?

File: pages/about/policies.html [hunk 1]
```diff
@@ -299,7 +299,8 @@ <h1 class="ui header">
                   Secondary School Literacy Policy
                 </h1>
                 <p>
-                    Preview Content goes here
+                    <strong><em>Secondary School Literacy Policy</em></strong> "Mount Sion CBS Secondary School is an inclusive boys&rsquo; only school which values the abilities and achievements of all its pupils. Our school accepts the fundamental principle that literacy is the key to improving learning and raising standards. The teaching of literacy is not the sole responsibility of the English Department; it is the responsibility of every teacher within the school. Literacy is one of the cornerstones of education and it enables pupils to gain access to the school curriculum, to read for information and pleasure, and to communicate effectively...
```
Comment (laurafitzgerald) [inline pages/about/policies.html]: @denispop9 when I load the screen this text takes up more space than the other sections. Can you make it to match the others?

Comment (denispop9) [reply pages/about/policies.html]: ye no problem

File: pages/about/policies.html [hunk 2]
```diff
@@ -287,7 +287,7 @@ <h1 class="ui header">
                     School Organisation
                 </h1>
                 <p>  
-                Preview Content goes here
+                <strong><em>School Organisation Policy</em></strong> "Mount Sion was founded by Edmund Rice in 1802. It was the first school founded by the Christian Brothers. The school has a proud tradition of educational service in Waterford and the surrounding areas...
```
Comment (laurafitzgerald) [inline pages/about/policies.html]: also @denispop9 I think this should be included as part of another pr?

Comment (denispop9) [reply pages/about/policies.html]: I'm going to do that right now:)

**Comment by denispop9** [comment]
@laurafitzgerald I think I am done could you check for me one last time?
